src/presenters/SalesFunnelFrontendPresenter.php

```diff
@@ -476,7 +476,7 @@ public function renderSubmit()
         // container items
         $paymentItemContainer = (new PaymentItemContainer())->addItems(SubscriptionTypePaymentItem::fromSubscriptionType($subscriptionType));
         if ($additionalAmount) {
-            $donationPaymentVat = $this->applicationConfig->get('donation_vat_rate');
+            $donationPaymentVat = (int)$this->applicationConfig->get('donation_vat_rate');
```

Comment:
I would probably cast this 4 lines later, as casting this to `int` here would make `$donationPaymentVat === null` always `false` - it would cast `null` to `0`.

Reply:
Yes, that would be safer, I moved it down exactly when it is called.